liboptv/src/orientation.c

```diff
@@ -141,20 +149,803 @@ double weighted_dumbbell_precision(vec2d** targets, int num_targs, int num_cams,
+    printf ("kappa          = %+8.4f deg    +/- %8.4f\n", cal->ext_par.kappa*RO, sigmabeta[5]*RO);
+    
+    
+    printf ("camera const   = %+8.3f mm     +/- %8.3f\n", cal->int_par.cc, sigmabeta[6]);
+    printf ("xh             = %+8.3f mm     +/- %8.3f\n", cal->int_par.xh, sigmabeta[7]);
+    printf ("yh             = %+8.3f mm     +/- %8.3f\n", cal->int_par.yh, sigmabeta[8]);
+    printf ("k1             = %+8.3f        +/- %8.3f\n", cal->added_par.k1, sigmabeta[9]);
+    printf ("k2             = %+8.3f        +/- %8.3f\n", cal->added_par.k2, sigmabeta[10]);
+    printf ("k3             = %+8.3f        +/- %8.3f\n", cal->added_par.k3, sigmabeta[11]);
+    printf ("p1             = %+8.3f        +/- %8.3f\n", cal->added_par.p1, sigmabeta[12]);
+    printf ("p2             = %+8.3f        +/- %8.3f\n", cal->added_par.p2, sigmabeta[13]);
+    printf ("scale for x'   = %+8.3f        +/- %8.3f\n", cal->added_par.scx, sigmabeta[14]);
+    printf ("shearing       = %+8.3f        +/- %8.3f\n", cal->added_par.she*RO, \
+                                                                    sigmabeta[15]*RO);
+                                                                    
+    if(flags->interfflag){
+    printf ("glass_x        = %8.3f mm      +/- %8.3f\n", cal->glass_par.vec_x/nGl, \
+                                                        (sigmabeta[16]+sigmabeta[17]));
+    printf ("glass_y        = %8.3f mm      +/- %8.3f\n", cal->glass_par.vec_y/nGl, \
+                                                        (sigmabeta[16]+sigmabeta[17]));
+    printf ("glass_z        = %8.3f mm      +/- %8.3f\n", cal->glass_par.vec_z/nGl, \
+                                                        (sigmabeta[16]+sigmabeta[17]));
+    }
+
+
+    /* this part is also irrelevant for liboptv, temporary saved to remind us
+       to check outside the orient() and act in the main program to deal with
+       the multi-plane or other type of calibration
+   */
+    /*
+    par_file = fopen_r ("parameters/examine.par");
+    fscanf (par_file,"%d\n", &dummy);
+    fscanf (par_file,"%d\n", &multi);
+    fclose (par_file);
+    if (dummy==1){
+      examine=4;
+    }
+    else{
+      examine=0;
+    }
+    */
+
+    /* show original images with residual vectors (requires globals)
+    is commented out as irrelevant but kept for the future move to the
+    Python side
+    */
+
+    /*
+    printf ("%d: %5.2f micron, ", nr+1, sigma0*1000);
+    printf("\ntest 1 inside orientation\n");
+    for (i=0; i<n_obs-10; i+=2)
+    {
+      n = pixnr[i/2];
+      intx1 = (int) pix[n].x;
+      inty1 = (int) pix[n].y;
+      intx2 = intx1 + resi[i]*5000;  // arbitrary elongation factor for plot
+      inty2 = inty1 + resi[i+1]*5000;
+    orient_x1[n] = intx1;
+    orient_y1[n] = inty1;
+    orient_x2[n] = intx2;
+    orient_y2[n] = inty2;
+    orient_n = n;
+    }
+
+    */
+
+
+
+    if (stopflag){
```

Comment:
We should change the error handling so that ``orient()`` returns an integer status: 1 for success, 0 for no convergence, etc.